ecl/hql/hqlmanifest.cpp

```diff
@@ -198,6 +198,27 @@ void ResourceManifest::loadResource(const char *filepath, MemoryBuffer &content)
     read(fio, 0, (size32_t) f->size(), content);
 }
 
+void getResourceFromJfrog(StringBuffer &localPath, IPropertyTree &item)
+{
+    StringBuffer filename(localPath);
+    getFileNameOnly(filename, false);
+
+    StringBuffer downloadPath;
+    getTempFilePath(downloadPath, "jfrog", nullptr);
+    downloadPath.append(filename);
+
+    StringBuffer jfrogCmd("jf rt dl --flat --quiet ");
```

Comment:
How is authentication handled?

Reply:
The server address and credentials are configured with the JFrog cli before making any calls to Artifactory. The connections from getResourceFromJfrog use the new configuration instead of having to pass the credentials at the command line.